content/content-tool/tool/src/webapp/vm/content/sakai_filepicker_attach.vm

```diff
@@ -43,6 +44,21 @@
 		#end	
 	</div>
 
+	<form name="doattachitem" style="display:none" id="doattachitem" action="#toolForm("FilePickerAction" "Main")" method=post enctype="multipart/form-data">
+	<input type="hidden" name="sakai_action" value="doAttachitem">
+	<input type="hidden" name="itemId" id="doattachitemid" />
+	<input type="hidden" name="sakai_csrf_token" value="$sakai_csrf_token" />
+	</form>
+
+	<script language="JavaScript" type="text/JavaScript">
```

Comment:
script language should no longer be used.... 

  http://stackoverflow.com/questions/2267476/html-script-tag-type-or-language-or-omit-both

Just <script type="text/javascript">